packages/office-ui-fabric-react/config/api-extractor.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compiler": {
+    "configType": "tsconfig",
+    "rootFolder": "./"
```

Comment:
Hmm... API Extractor should probably support the "extends" field, so you can have individual **api-extractor.json** files but share the common configuration.  This is probably pretty easy to implement.  Consider opening a GitHub issue here:  [https://github.com/Microsoft/web-build-tools/issues](https://github.com/Microsoft/web-build-tools/issues)

Reply:
Opened issue: https://github.com/Microsoft/web-build-tools/issues/899